src/lisp/kernel/clos/print.lisp

```diff
@@ -280,6 +280,23 @@ printer and we should rather use MAKE-LOAD-FORM."
     (write-string ")" stream)
     obj))
 
+#-staging
+(defmethod print-object ((l list) stream)
+  (if (cdr l)
+      (case (first l)
+        (eclector.reader:quasiquote
+         (write-char #\` stream)
+         (core:write-object (second l) stream))
+        (eclector.reader:unquote
+         (write-char #\, stream)
+         (core:write-object (second l) stream))
+        (eclector.reader:unquote-splicing
+         (write-string ",@" stream)
+         (core:write-object (second l) stream))
+        (otherwise
+         (call-next-method)))
```

Comment:
Is it okay for this method to be on `list`? If we're really using `print-object` pervasively, would there not be methods on `cons` and `null` that take priority? Presumably we don't actually have such methods, if this works, but that's the organization I'd imagine